File: listacasa3.py/ex10.py
while True:
     
     op = (input("Entre com a operação (+, -, /, *, S):"))
     a = float(input("Digite o primeiro valor:"))
     b = float(input("Digite o segundo valor:"))
     resultado = 0
     
     if op == 'S':
        print("Ok, Saindo do progama.")
        break
     
     if op == '+':
         resultado = a + b
         print("Resultado", resultado)
     elif op == '-':
         resultado = a * b  
         print("Resultado", resultado)
     elif op == '*':
         resultado = a * b  
         print("Resultado", resultado)
     elif op == '/':
         if a and b != 0:
             resultado = a / b
             print("Resultado", resultado)
         else:
             print("Não é permitido divisão com zero:")
     else:
         print("Operação invalida.") 

File: listacasa3.c/ex10.c
#include <stdio.h>

int main() {
    char operacao;
    int a, b;
    do {
        printf("Digite uma operacao ('+', '-', '*', '/', 'S' para sair): ");
        scanf(" %c", &operacao);

        if (operacao == 'S' || operacao == 's') {
            break;
        }

        printf("Digite o primeiro numero: ");
        scanf("%d", &a);
        printf("Digite o segundo numero: ");
        scanf("%d", &b);

        switch (operacao) {
            case '+':
                printf("Resultado: %d + %d = %d\n", a, b, a + b);
                break;
            case '-':
                printf("Resultado: %d - %d = %d\n", a, b, a - b);
                break;
            case '*':
                printf("Resultado: %d * %d = %d\n", a, b, a * b);
                break;
            case '/':
                if (b != 0) {
                    printf("Resultado: %d / %d = %d\n", a, b, a / b);
                } else {
                    printf("Erro: Divisao por zero!\n");
                }
                break;
            default:
                printf("Operacao invalida!\n");
        }

    } while (operacao != 'S' && operacao != 's');

    return 0;
}
